uber/model_checks.py

```diff
@@ -166,22 +166,6 @@ def group_money(group):
         except Exception:
             return "What you entered for Total Group Price ({}) isn't even a number".format(group.cost)
 
-    try:
```

Comment:
Admins can no longer set amount_paid and amount_refunded directly, so these checks aren't needed.